white-paper.md

```diff
@@ -148,7 +148,7 @@ The submission of a package release requires multiple transactions to occur atom
 
 Failure of any one of the three operations will result in the protocol reverting to its previous state, thus eliminating any evidence of the submission.
 
-When a package is successfully submitted, the package maintainer will receive a maintainer NFT to evidence their work and contribution to open-source. The package maintainer may transfer the steeping rewards associated with the maintainer NFT to a third party. However, the reputation associated with the creation and maintenance of the asset will remain with the package maintainer, so their reputation can be affected over time. As the reputation of any member of the tea community reaches key milestones, they may be granted access to elevated parts of the protocol or receive accelerated rewards, as decided by the tea governance. For more details on the maintainer NFT, see the maintainer NFT section.
+When a package is successfully submitted, the package maintainer will receive a maintainer NFT to evidence their work and contribution to open-source. The package maintainer may transfer the steeping rewards associated with the maintainer NFT to a third party. However, the reputation associated with the creation and maintenance of the asset will remain with the package maintainer, so their reputation can be affected over time. As the reputation of any member of the tea community reaches key milestones, they may be granted access to elevated parts of the protocol or receive accelerated rewards, as decided by the tea governance. For more details on the maintainer NFT, see the maintainers NFT section.
```

Comment:
The section is called “maintainer NFT” so adding an “s” would refer to a section that doesn’t exist.